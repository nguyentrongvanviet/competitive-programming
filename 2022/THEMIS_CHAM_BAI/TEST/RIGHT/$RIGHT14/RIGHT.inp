5 3
3
1
5
2
4
