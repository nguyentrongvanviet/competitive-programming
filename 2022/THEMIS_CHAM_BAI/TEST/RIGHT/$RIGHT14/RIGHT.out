6
4
11
8
13
